File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
#include <vector>

class Solution {
public:
    int removeDuplicates(std::vector<int>& nums) {
        int size = nums.size();
        if (size == 0) {
            return 0;
        }

        int uniqueIndex = 0;
        for (int i = 1; i < size; i++) {
            if (nums[i] != nums[uniqueIndex]) {
                uniqueIndex++;
                nums[uniqueIndex] = nums[i];
            }
        }

        return uniqueIndex + 1;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if (nums.length === 0) {
            return 0;
        }

        let uniqueIndex = 0;
        for (let i = 1; i < nums.length; i++) {
            if (nums[i] !== nums[uniqueIndex]) {
                uniqueIndex++;
                nums[uniqueIndex] = nums[i];
            }
        }

        return uniqueIndex + 1;
};